test/test_jit.py

```diff
@@ -213,7 +213,7 @@ def allSum(vs):
             for g2, g2_ge in zip(grads2, grads2_ge):
                 if g2 is None and g2_ge is None:
                     continue
-                self.assertTrue(torch.allclose(g2, g2_ge, atol=5e-4, rtol=1e-4))
+                self.assertTrue(torch.allclose(g2, g2_ge, atol=7e-4, rtol=1e-4))
```

Comment:
oh? What happened here?

Reply:
test_lstm_fusion_concat has precision problems. I believe Roy has previously had problems with this test on his machine, so it has likely always been right on the edge.